stack/conda.rst

```diff
@@ -24,6 +24,48 @@ Note that each of these other environments is maintained separately; see below.
 Requests for additions to ``rubin-env`` should be made via RFC.
 The DM-CCB must approve such requests.
 
+.. _rubin-env-switching-versions:
+
+Switching rubin-env versions
+----------------------------
+
+There might be an instance where you might need to change the current version of rubin-env.
+You can view the current available env `here <https://anaconda.org/conda-forge/rubin-env/files>`__
+To see all available ``rubin-env`` in your terminal run the following:
+
+.. code-block:: bash
+
+   conda search -c nodefaults  -c conda-forge rubin-env
+
+This command will print out only production ready ``rubin-env``.
+To change the ``rubin-env``:
+
+.. code-block:: bash
+
+   git clone https://github.com/lsst/lsstsw
+   cd lsstsw
+   LSST_CONDA_CHANNELS="nodefaults conda-forge"
+   ./bin/deploy -v 8.0.0 # or any other rubin env
```

Comment:
I'm sorry for not being clear. I know it can be run repeatedly, I was trying to ask if we've documented clearly on this page that you can run it as often as you want.

Reply:
Oh, okay. It is not documented clearly on this page, but is this the right page? When I joined, I assumed it was okay to rerun that command because I saw it being used in a few other places like https://arc.net/l/quote/atptszyd. I think it would be better to mention it in The LSST Software Build Tool section.

Reply:
That link goes to a section that I don't think is still valid information now that we install EUPS via conda-forge. I think the whole section about setting eups versions can be removed because you get the eups version you get from conda-forge and if you want another version you use conda to change it. In the past eups was not on conda-forge and had to be installed separately by deploy.

Also, [that page](https://developer.lsst.io/stack/lsstsw.html#introduction) does indeed say that deploy can be run multiple times so my question is answered.